src/css/SocialsIcon.module.css

```diff
@@ -3,17 +3,24 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  width: 129px;
+  width: max-content;
 }
 
-.iconLink:hover {
-  opacity: 0.95;
+.iconLink {
+  margin-right: 0.75rem;
 }
 
-.img {
-  opacity: 1;
+.iconLink:focus {
+  outline: none;
```

Comment:
I removed `outline` on focus state of the social media link because it looks...more aesthetic visually? As we do have [another visual cue applied for focus state](https://github.com/sfusurge/surge_website/pull/34/files#diff-fd82ec4037ce1ddb81882f8e497f237d8bf73270e77c6112ddb5d9493bc62cf0R18), people who navigate the page using the keyboard can still notice the (subtle) change as they reach the social media link, so accessibility is still doing somewhat okay here. I'm not sure, though, if those who use keyboard to navigate will actually expect to see the outline because that's the pattern that they learn from using other websites before. 🤔 How do ya think? @eth-p 

<img width="326" alt="Screen Shot 2022-08-12 at 21 19 49" src="https://user-images.githubusercontent.com/65995018/184468346-a46242e5-9626-46b8-bc1c-fe0e279fdf76.png">

<img width="318" alt="Screen Shot 2022-08-12 at 21 19 56" src="https://user-images.githubusercontent.com/65995018/184468348-76fda370-30da-4e58-940d-a488503ec591.png">

Reply:
Speaking of focus state, as I navigate the website using the keyboard, I realized a lot of elements are missing focus state though 😅 We should probably fix them some time in the future haha